.github/workflows/ci.yml

```diff
@@ -309,8 +309,6 @@ jobs:
         with:
           python-version: ${{ needs.build-info.outputs.defaultPythonVersion }}
         if: needs.build-info.outputs.inWorkflowBuild == 'true'
-      - run: ./scripts/ci/install_breeze.sh
-        if: needs.build-info.outputs.inWorkflowBuild == 'true'
       - name: "Retrieve DEFAULTS from the _initialization.sh"
```

Comment:
I think we can remove that one entirely. It's copied from `build.yml` but it is only needed there. We only do that in `build.yml` to make sure that we will use values defined in the PR and not in main (build.yml by default uses workflow and scripts from main and not from PR for security reasons). Actually I plan to remote that part completely once we convert all the scripts to Python as in Python it can (and will be) done much easier so maybe let's not touch it :).

Reply:
Sure, let's not remove it now. Thoughts on reordering it so it's in a more consistent order between CI and PROD?

Reply:
Absolutely. There is no bad side effect of it.